src/app/shared/services/chart-seizures-by-reason.service.ts

```
@@ -0,0 +1,43 @@
+      .listCollection([
+        orderBy('occurred', 'desc'),
+        where('occurred', '>=', this.dateFrom.toDate()),
+        where('occurred', '<=', this.dateTo.toDate()),
+      ])
+      .pipe(map((seizures) => this.aggregateSeizuresByReason(seizures)));
+  }
+
+  private aggregateSeizuresByReason(seizures: Seizure[]): ChartData {
+    const reasonCounts: { [reason: string]: number } = {};
+    for (const seizure of seizures) {
+      if (seizure.triggers && seizure.triggers.length) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)